code/modules/mob/living/simple_animal/abnormality/teth/clayman.dm

```diff
@@ -0,0 +1,74 @@
+/mob/living/simple_animal/hostile/abnormality/clayman
+	name = "Generic brand modelling clay"
+	desc = "A small, rough humanoid figure made of clay."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "bluro"
+	icon_living = "bluro"
+	icon_dead = "bluro"
+	portrait = "bluro"
+	del_on_death = TRUE
+	maxHealth = 1000
+	health = 1000
+	ranged = TRUE
+	rapid_melee = 1
+	melee_queue_distance = 2
+	move_to_delay = 3
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1, PALE_DAMAGE = 1)
+	melee_damage_lower = 10
+	melee_damage_upper = 12
+	melee_damage_type = RED_DAMAGE
+	attack_sound = 'sound/effects/hit_kick.ogg'
+	attack_verb_continuous = "smashes"
+	attack_verb_simple = "smash"
+	friendly_verb_continuous = "bonks"
+	friendly_verb_simple = "bonk"
+	can_breach = TRUE
+	threat_level = TETH_LEVEL
+	start_qliphoth = 1
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(50, 50, 50, 50, 50),
+		ABNORMALITY_WORK_INSIGHT = list(50, 50, 50, 50, 50),
+		ABNORMALITY_WORK_ATTACHMENT = list(50, 50, 50, 50, 50),
+		ABNORMALITY_WORK_REPRESSION = list(50, 50, 50, 50, 50),
+	)
+	work_damage_amount = 0
+	work_damage_type = RED_DAMAGE
+	death_message = "loses form."
+	ego_list = list(
+		/datum/ego_datum/weapon/clayman,
+		/datum/ego_datum/armor/clayman
```

Comment:
You nearly got this one perfectly, work_chances is made good but this one is missing a " , " at the end
When doing multi-line lists all objects should have it at the end so future github additions only show 1 line changed instead of 2
```suggestion
		/datum/ego_datum/armor/clayman,
```